vector-im/element-android
# Voice Broadcast - Send state events

## Type of change


## Content

- Add the `io.element.voice_broadcast_info` state event related to https://github.com/vector-im/element-meta/discussions/632
- Add use cases to play/pause/resume/stop a voice broadcast (by just sending the correct state events, without recording for the moment)
- Add a minimalist view in the timeline related to this state event

## Motivation and context

Continue #7127 

## Screenshots / GIFs

![image](https://user-images.githubusercontent.com/11990514/193525739-f9502e50-12d4-4ace-911f-2bbcb1d1a23e.png)

## Tests

- Enable the voice broadcast feature flag
- Start a voice broadcast from the message composer actions
- Verify that a widget is displayed in the timeline
- Play with the widget buttons and verify that the voice broadcast state is correctly updated

## Tested devices

- OS version(s):

## Checklist
